docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -500,6 +500,9 @@ following properties:
 * - `iceberg.rest-catalog.vended-credentials-enabled`
   - Use credentials provided by the REST backend for file system access.
     Defaults to `false`.
+* - `iceberg.rest-catalog.support-nested-namespace`
```

Comment:
We usually don't use `support-` prefix for boolean config property. It should be `iceberg.rest-catalog.nested-namespace-enabled` or something. I would recommend looking for similar property names when you adding a new property. 

However, I think character is better than boolean because Iceberg may make it configurable. https://github.com/apache/iceberg/pull/10877

Reply:
thanks for clarifying. I renamed it.
Should we can make it configurable to take character once the mentioned PR lands?